src/sentry/models/pullrequest.py

```diff
@@ -91,17 +91,31 @@ class Meta:
         unique_together = (("pull_request", "commit"),)
 
 
+class CommentType:
+    MERGED_PR = 0
+    OPEN_PR = 1
+
+    @classmethod
+    def as_choices(cls) -> Sequence[Tuple[int, str]]:
+        return ((cls.MERGED_PR, "merged_pr"), (cls.OPEN_PR, "open_pr"))
+
+
 @region_silo_only_model
 class PullRequestComment(Model):
     __include_in_export__ = False
 
     external_id = BoundedBigIntegerField()
-    pull_request = FlexibleForeignKey("sentry.PullRequest", unique=True)
+    pull_request = FlexibleForeignKey("sentry.PullRequest")
     created_at = models.DateTimeField()
     updated_at = models.DateTimeField()
     group_ids = DjangoArrayField(BoundedBigIntegerField())
     reactions = JSONField(null=True)
+    comment_type = BoundedPositiveIntegerField(
+        default=CommentType.MERGED_PR,
+        choices=CommentType.as_choices(),
```

Comment:
You should also have null=False in this field definition so that it matches the schema you're creating. Without the null=False parameter, the schema generated for tests will lack `NOT NULL`.